gdx/src/com/badlogic/gdx/math/Intersector.java

```diff
@@ -92,24 +96,36 @@ public static boolean intersectSegmentPlane (Vector3 start, Vector3 end, Plane p
 		return true;
 	}
 
-	/** Determines on which side of the given line the point is. Returns -1 if the point is on the left side of the line, 0 if the
-	 * point is on the line and 1 if the point is on the right side of the line. Left and right are relative to the lines direction
-	 * which is linePoint1 to linePoint2. */
+	/**
+	 * Determines on which side of the given line the point is.<p>
+	 * Left and right are relative to the line's direction which is linePoint1 to linePoint2.
+	 * 
```

Comment:
I am not usually a fan of blank lines in javadocs. When they are really needed I use a `<p>` on the line by itself (which is what you will get if you ran the formatter).

Reply:
</p> is not needed at all.

Reply:
In fact, I've had trouble in the past trying to use `</br>` in JavaDoc in the past, so I now avoid using closed tags (such as `</p>`).